.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+
+addons:
+  apt:
+    packages:
+      - texlive-latex-base
+      - texlive-fonts-recommended
+
+install:
+  - wget -O bibutils_5.6_src.tgz https://sourceforge.net/projects/bibutils/files/bibutils_5.6_src.tgz/download
```

Comment:
not needed anymore, as we can use apt's package bibutils.  however that is an older version; need to retest everything.  See https://github.com/collective/bibliograph.parsing/pull/4